Jropp/jason-ropp-js-iobp
# Polymer Upgrade: #1 Set properties

## What It Does

This batches property changes using this.setProperties()

Batching the property changes means that any property effects are set once rather than once for each property you set.

Any functions with two or more properties being set were batched using this.setProperties

```js
this.cow = "moo";
this.pig = "oink";

// CHANGED TO

this.setProperties({
  cow: "moo",
  pig: "oink"
}
```

## How To Test

All functionality should be the same as before. Create, delete, edit users. Sort.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

- [ ] #167
